pkg/cluster/fetch.go

```diff
@@ -41,10 +41,11 @@ type ManagementComponents struct {
 // that indicates an upgrade is required. In the future, we might change the bundles API to remove the assumption
 // and make this explicit. When that happens, this method will need to change.
 func ManagementComponentsFromBundles(bundles *v1alpha1release.Bundles) *ManagementComponents {
-	return newManagementComponents(&bundles.Spec.VersionsBundles[0])
+	return NewManagementComponents(&bundles.Spec.VersionsBundles[0])
 }
 
-func newManagementComponents(vb *v1alpha1release.VersionsBundle) *ManagementComponents {
+// NewManagementComponents returns a ManagementComponents object built from a VersionsBundle.
+func NewManagementComponents(vb *v1alpha1release.VersionsBundle) *ManagementComponents {
```

Comment:
We're only exporting this for now to convert the VersionsBundle returned from spec.RootsVersionsBundle() into a `ManagementComponents` struct. Once, the CAPI upgrader is changed to use management components for the `Upgrade`, we won't need to export this anymore.